service-base/src/main/java/fi/nls/oskari/domain/map/wfs/WFSLayerAttributes.java

```diff
@@ -123,7 +123,7 @@ public boolean hasFilter() {
 
     public List<String> getSelectedAttributes(String lang) {
         return params.getOrDefault(lang,
-                params.getOrDefault(PropertyUtil.getDefaultLanguage(), Collections.emptyList()));
+                params.getOrDefault("default", Collections.emptyList()));
```

Comment:
For using this we wouldn't need to do anything for the frontend if we just use the instances default language (Could use Oskari.getLocalized() in the frontend). If we change this we will need to add handling for the "default".

Reply:
Reverted